db/migrate/20140114144500_signin_permissions_must_be_lowercase.rb

```diff
@@ -1,5 +1,6 @@
 class SigninPermissionsMustBeLowercase < ActiveRecord::Migration
   def up
-    SupportedPermission.update_all({ name: 'signin' }, { name: 'Signin' })
+    SupportedPermission.where({ name: 'Signin' })
```

Comment:
May as well remove the unnecessary braces around the options as well, if we're tidying this up. :+1: